src/common/buttons/underline-button.tsx

```diff
@@ -0,0 +1,33 @@
+import { classnames } from 'helpers/utils';
+import React, { ReactNode } from 'react';
+import styles from './underline-button.module.scss';
+
+const DefaultProps = {
+  onClick: undefined,
+  isDisabled: undefined,
+  isActive: undefined,
+};
+
+interface ButtonProps {
+  children: ReactNode;
+  onClick?: () => void;
+  isDisabled?: boolean;
+  isActive?: boolean;
+}
+
+const UnderlineButton = (props : ButtonProps) => (
+  <button
+    className={classnames(
+      !props.isActive ? styles.customUnderlineButtonInactive : styles.customUnderlineButton,
+    )}
+    type="button"
+    onClick={props.onClick}
+    disabled={props.isDisabled}
+  >
+    {props.children}
+  </button>
+);
+
+UnderlineButton.defaultProps = DefaultProps;
+
+export { UnderlineButton };
```

Comment:
So, this is more of a discussion. Looking at this component it looks very similar to the `./button.tsx` one (code-wise).
What that makes me wanna do is:

1.  Allow the other `Button` to accept classes as a prop, loading them with the `classnames` helper
2. Keep this component, but render the original `Button`. This component would only have the `isActive` prop and would pass the classes (it would still technically have the others, but we wouldn't really use them here, we would pass them down to Button directly and extend the interface).

What do you think of this approach? We can discuss this further if you want, or I can help you out if you have questions.

To pass down the props you could do:

```
<Button
  className={...} // or however you call the prop that receives classes in `Button`
  {...props} // rest of the props would be destructured, including {children}
/>
```